src/models/backend/transfers.rs

```diff
@@ -39,13 +40,29 @@ pub struct Erc20Transfer {
     pub block_number: u64,
     pub transaction_hash: String,
     pub to: String,
+    #[serde(deserialize_with = "default_if_null")]
     pub value: String,
     pub token_address: String,
     #[derivative(Hash = "ignore")]
     pub token_info: Option<TokenInfo>,
     pub from: String,
 }
 
+#[derive(Derivative, Deserialize, Debug, Clone)]
+#[derivative(Hash)]
+#[serde(rename_all = "camelCase")]
+pub struct Erc20TransferMalformed {
```

Comment:
Not a big fan of this :/

Reply:
Oh wait, that's totally unnecessary actually, just forgot to clean up.